source/includes/_010-introduction.md

```diff
@@ -8,7 +8,7 @@ All APIs SHOULD be accessed from `https://go-server-url:8154/go/api`. All data S
 
 ## Current version
 
-By default, all requests receive **Version 1** of the API. It is recommended to explicitly request this version via the `Accept` header.
+Since v15.2.0,  all requests receive **Version 1** of the API. It is recommended to explicitly request this version via the `Accept` header.
```

Comment:
No the new APIs explicitly require a header. We can make them work without one, but that makes it ambiguous when we have to serve 2 different API versions at the same URL.